src/web100srv.c

```diff
@@ -1721,39 +1637,32 @@ void NDT_server_main_loop(SSL_CTX *ssl_context, int listenfd) {
         continue;
       }
 
-      /* if ((mchild->next == NULL) && (mchild->running == 0))
-       *   mchild = tmp_ptr;
-       * if (mchild != head_ptr) 
-       */
       tmp_ptr = mchild;
 
       // Update queued clients, send message to client when it moves
       //  up in the queue enough to get closer to running a test.  This
       //  happens when the client falls into the next lower maxqueue bin
-      //  RAC 3/21/10
 
-      int rac;
       if (waiting > (2 * max_clients)) {
+        int wait_data = -1;
+        int num_secs = 0;
```

Comment:
I don't understand. The log message claims that it sent the information someplace but it didn't.